frontend/packages/console-shared/src/selectors/common.ts

```diff
@@ -32,3 +32,7 @@ export const getAnnotations = <A extends K8sResourceCommon = K8sResourceCommon>(
   value: A,
   defaultValue?: K8sResourceCommon['metadata']['annotations'],
 ) => (_.has(value, 'metadata.annotations') ? value.metadata.annotations : defaultValue);
+export const getTaints = <A extends K8sResourceKind = K8sResourceKind>(
+  value: A,
+  defaultValue?: Toleration[],
+) => (_.has(value, 'spec.taints') ? value.spec.taints : defaultValue);
```

Comment:
TS optional chaining is preferred to lodash

Reply:
fixed.